### DISCUSSION TITLE
Caching requests unrelated to models

### DISCUSSION THREAD

**Discussion by stafyniaksacha** [strapi-community/strapi-plugin-rest-cache#21] [Ideas]
Today, this plugin only handle routes that are related to a content type. 

We can introduce caching for other requests and decide programmatically when they are invalidated.

My questions are:
- What would be the configuration for this?
- How would we handle automatic purging to avoid stale data?
- What can be the tests cases?
- Do someone want to lead this topic?

**Comment by oDinZu** [comment]
I like the idea of adding caching for other assets; personally for my use case is with Jekyll  + Strapi. 

I want to cache the images locally for content-type(s)  to then build the website without having to ping the Strapi api server on each build.
